AirCasting/SessionViews/StandaloneSessionCardView.swift

```diff
@@ -58,6 +59,10 @@ struct StandaloneSessionCardView: View {
     var finishAndSyncButton: some View {
         Button(Strings.StandaloneSessionCardView.finishAndSyncButtonLabel) {
             if networkChecker.connectionAvailable {
+                guard !userSettings.syncOnlyThroughWifi || networkChecker.isUsingWifi else {
+                    alert = InAppAlerts.noWifiNetworkSyncAlert()
+                    return
+                }
```

Comment:
don't we want a no-connection alert too?

btw. I'll investigate possibilities to unify this and not repeat in each view

Reply:
There is an alert for no connection in line 70